core/src/test/scala/org/apache/spark/sql/delta/DeltaWriteConfigsSuite.scala

```diff
@@ -365,21 +365,18 @@ class DeltaWriteConfigsSuite extends QueryTest
                 // Specified schema is missing field(s): bar
                 // Specified schema has additional field(s): foo
                 assert(outputLoc == "path" && outputMode == "c_or_r_replace")
-                dtb_output += ((outputLoc, outputMode, false, false, false, true, ""))
+                dtb_output += ((outputLoc, outputMode, false, false, true, ""))
               case _ =>
                 val config = log.snapshot.metadata.configuration
 
-                val answer_no_prefix_lowercase =
-                  config.contains(config_no_prefix.toLowerCase(Locale.ROOT))
                 val answer_no_prefix = config.contains(config_no_prefix)
                 val answer_prefix = config.contains(config_prefix)
 
-                assert(answer_no_prefix_lowercase) // bizarre!
-                assert(!answer_no_prefix)
+                assert(answer_no_prefix) // bizarre!
```

Comment:
delete the `bizarre` comment

Reply:
and the one below. this is expected behaviour.

Reply:
I found it a bizarre comment 🤣